content/en/docs/reference/kubectl/kubectl.md

```diff
@@ -377,6 +377,14 @@ kubectl [flags]
 </td>
 </tr>
 
+<tr>
+<td colspan="2">KUBECTL_DEBUG_CUSTOM_PROFILE</td>
```

Comment:
Thanks, but: I think we prefer these variables in alphabetical order. Could you make them sorted? (I can see they are already close).

Reply:
You are right, moved it to correct place in alphabetical order.